tup.config.default

```diff
@@ -0,0 +1,6 @@
+CONFIG_DEBUG=y
+CONFIG_BOARD_VERSION=v3.3
+CONFIG_BOARD_48V=n
```

Comment:
High voltage version might not always be 48V, I am already thinking of increasing it to 60ish V. So better call it HV.

Reply:
Make sure to also update readme.